spec/identity/index.html

```diff
@@ -453,6 +453,14 @@ <h2 id="h2_introduction"><span class="secno">1. </span>Introduction</h2><p><em>T
 WebIDs can be used to build a Web of trust using vocabularies such as FOAF [<cite><a class="bibref" href="#bib-FOAF">FOAF</a></cite>] by allowing people to link their profiles in a public or protected manner.
 Such a web of trust can then be used by a <a class="tref internalDFN" title="Service" href="#dfn-service">Service</a> to make authorization decisions, by allowing access to resources depending on the properties of an agent, such as that they are known by some relevant people, employed at a given company, a member of a family or some other group, etc.
 </p>
+<p>This specification is for:</p>
+<ul>
+  <li>Anyone that wants to understand the architectural principles and notions underlying WebID.</li>
+  <li>URI owners who want to allocate identifiers to denote agents.</li>
+  <li>Resource server developers who want to enable clients to consume WebIDs and their associated descriptions.</li>
+  <li>Application developers who want to implement a client to consume WebIDs.</li>
+  <li>Specification authors who want to integrate or extend WebID.</li>
+</ul>
```

Comment:
Applying @kidehen's suggested tweaks, with some tweaks of my own
```suggestion
  <li>Anyone who wants to understand the architectural principles and notions underlying WebID</li>
  <li>Content publishers who want to allocate identifiers to denote agents, unambiguously</li>
  <li>Server application developers who want to provide client applications with WebIDs and associated agent descriptions</li>
  <li>Client application developers who want to use a WebID to unambiguously identify their applications as agents</li>
  <li>Specification authors who want to extend WebID via additional specifications, e.g., identity authentication protocols</li>
</ul>
```